File: examples/js/allin-cancel-spot-order.js
import ccxt from '../../js/ccxt.js';
// AUTO-TRANSPILE //
async function example() {
    const exchange = new ccxt.allin({
        'apiKey': 'MG9pDxDRRRJZkuFDAVK6RZyiexsXKN',
        'secret': 'jN2j4gnUPyaOD3wVDoOZpyq5s8ymCgYN4khCwp2aEnsOf4W1na',
        'options': {
            'defaultType': 'spot',
        },
    });
    exchange.setSandboxMode(true);
    await exchange.loadMarkets();
    exchange.verbose = true; // uncomment for debugging purposes if necessary
    const orders = await exchange.cancelOrder('38', 'BTC/USDT', {});
    console.log(orders);
}
await example();

File: examples/py/allin-cancel-spot-order.py
# -*- coding: utf-8 -*-

import asyncio
import os
import sys
from pprint import pprint

root = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
sys.path.append(root + '/python')

import ccxt.async_support as ccxt  # noqa: E402


async def example():
    exchange = ccxt.allin({
        "apiKey": "6jAuGb4cCulmdaxx2D8uCBfwZE400g",
        "secret": "8fh5YmDr01IXeq4xqWpQAGf6u2UQZynpWGJwz7cODKnf0QFNvt",
        "options": {
            "defaultType": "future"
        }
    })
    exchange.set_sandbox_mode(True)
    await exchange.load_markets()
    exchange.verbose = True  # uncomment for debugging purposes if necessary
    orders = await exchange.cancel_order('8381526', 'BTC/USDT', {})
    print(orders)

    await exchange.close()


asyncio.run(example())
